sdk/appconfiguration/azure-appconfiguration/tests/test_azure_configuration_client_aad.py

```diff
@@ -389,15 +392,15 @@ def test_list_revisions_key_label(self):
         assert cnt >= 2
 
     def test_list_revisions_only_label(self):
-        items = self.app_config_client.list_revisions(labels=[LABEL])
+        items = self.app_config_client.list_revisions(labels=LABEL)
         cnt = 0
         for kv in items:
             assert kv.label == LABEL
             cnt += 1
         assert cnt >= 1
```

Comment:
All your changes LGTM (I'm assuming there are corresponding swagger changes for the models args?) My only comment is noticing the following test pattern which I think could be expressed more clearly/concisely if you are felt like updating them:
```
items = list(self.app_config_client.list_revisions(labels=LABEL))
assert len(items) >= 1
assert all(x.label == LABEL for x in items)
```